features/resource-size.yml

```
@@ -0,0 +1,11 @@
+name: Resource size
+description: The `decodedSize`, `encodedSize`, and `transferSize` properties of the `PerformanceResourceTiming` API reports the size, in octets, of resources loaded.
```

```suggestion
description: The `decodedSize`, `encodedSize`, and `transferSize` properties of the `PerformanceResourceTiming` API reports the size, in bytes, of resources loaded.
```

We can also just not say what the unit is:

```suggestion
description: The `decodedSize`, `encodedSize`, and `transferSize` properties of the `PerformanceResourceTiming` API reports the size of resources loaded.
```

---

I cribbed the octets thing from MDN—I thought it was weirdly specific and thus significant. But if it's not, let's go with the unitless version.